docs/concept/buckconfig.soy

```diff
@@ -944,6 +945,23 @@ $ buck targets --resolve-alias app#src_jar
 {/call}
 
 {call buckconfig.section}
+  {param name: 'groovy' /}
+  {param description}
+    This section configures the <a href="http://groovy-lang.org/">Groovy</a> toolchain.
+  {/param}
+{/call}
+
+{call buckconfig.entry}
+  {param section: 'groovy' /}
+  {param name: 'groovy_home' /}
+  {param description}
+    This defines the value of <code>GROOVY_HOME</code> that Buck should use. If it is not provided,
+    Buck will use the system's <code>GROOVY_HOME</code> by default.
+  {/param}
+  {param example_value: '/path/to/groovy_home'/}
```

Comment:
Can you plop this on top of the description please to be consistent with the rest of the file? :)